airbyte-integrations/connectors/source-yandex-metrica/source_yandex_metrica/spec.yaml

```diff
@@ -0,0 +1,292 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Yandex Metrica Spec
+  type: object
+  required:
+    - auth_token
+    - counter_id
+    - start_date
+    - end_date    
+    - hits_fields
+    - visits_fields
+  additionalProperties: false
+  properties:
+    auth_token:
+      type: string
+      title: Authentication Token
+      description: Your Yandex Metrica API access token
+      airbyte_secret: true
+      order: 0
+    counter_id:
+      type: string
+      title: Counter ID
+      description: Counter ID
+      pattern: ^[0-9]+$
+      order: 1
+    start_date:
+      title: Start Date
+      type: string
+      description: UTC date and time in the format YYYY-MM-DD.
+      pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
+      examples:
+        - "2022-01-01"
+      order: 2
+    end_date:
+      title: End Date
+      type: string
+      description: UTC date and time in the format YYYY-MM-DD.
+      pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
+      examples:
+        - "2022-01-01"
+      order: 3
+    hits_fields:
+      title: Views Fields
+      description: Fields for page views data. See https://yandex.com/dev/metrika/doc/api2/logs/fields/hits.html
+      type: array
+      items:
+        type: string
+        enum:
+          - ym:pv:watchID
+          - ym:pv:counterID
+          - ym:pv:date
+          - ym:pv:dateTime
+          - ym:pv:title
+          - ym:pv:URL
+          - ym:pv:referer
+          - ym:pv:UTMCampaign
+          - ym:pv:UTMContent
+          - ym:pv:UTMMedium
+          - ym:pv:UTMSource
+          - ym:pv:UTMTerm
+          - ym:pv:browser
+          - ym:pv:browserMajorVersion
+          - ym:pv:browserMinorVersion
```

Comment:
Why not retrieve the common fields or a common report? I don't think this is the best way to retrieve it.

Reply:
Removed field choices during setup. Fields are passed from a dictionary in fields.py